README.md

```diff
@@ -55,3 +61,56 @@ If your proposal was closed because a core contributor determined that it was
 not worth pursuing and you feel that it was wrongly closed, then feel free
 to join the developer IRC group and have a more in-depth discussion with 
 other core developers about the feature.
+
+## How core developers evaluate proposals
+
+The following is a list of considerations that core developers use when deciding
+to accept, close, or leave a proposal open. It is intended to be useful for core
+developers when considering proposals and for proposal-makers in drafting their
+proposals.
+
+**1. Does the proposal comply with the rules?**
+Read the proposal and check to see that it complies with the above-stated rules.
+If it does not, close the proposal.
+
+**2. How much support is the proposal receiving?**
+Evaluate the amount of support the proposal is receiving. This is an ongoing
+analysis. If a proposal receives little support at first, it may receive
+additional support later on.
+
+**3. Can this proposal be implemented with an addon?**
+Evaluate whether it is possible for the proposal to be implemented inan addon.
+If it is possible for the proposal to be in an addon it is unlikely to be
+accepted.
+
+**4. Does this proposal benefit most users?**
+Determine whether this proposal benefits all users, or just certain users.
+For example, a feature that can only be used for 3D FPS games is less
+likely to be accepted than a feature that benefits all 3D games.
+
+**5. Can this proposal be implemented in a robust, general-purpose way?**
+Determine whether the feature can be implemented in a robust way that benefits
+all use-cases. For example, many games use an inventory system, however,
+every game implements inventory differently. Accordingly, a proposal for an
+inventory system will likely not be accepted because it would be impossible for
+us to implement an inventory system that works for most users that need an
+inventory in their game.
+
+**6. Does this proposal help users overcome a limitation?**
+Proposals that overcome a specific limitation are more likely to be accepted
+than proposals that are just helpful. In short, need-to-have features will be
+prioritized over nice-to-have features.
+
+**7. How complex would the proposed feature be?**
+A highly complex new feature involving substantial changes to core is less likely
+to be accepted than a feature that can be contained within a single node, or a
+group of nodes.
+
+**8. Can the feature be worked around in script with a few lines?**
+If the feature is intended to save users a few lines of code it is unlikely
+to be accepted.
+
+The above considerations are all balanced, no one is more important than another.
+Core developers have discretion to weigh the factors as they see fit.
+
+In addition to the above guideline consider The graph in [this article](https://godotengine.org/article/will-your-contribution-be-merged-heres-how-tell) which outlines what core developers consider when evaluating PRs.
```

Comment:
```suggestion
In addition to the above guideline, consider the graph in [this article](https://godotengine.org/article/will-your-contribution-be-merged-heres-how-tell) which outlines what core developers consider when evaluating PRs.
```

Reply:
Not fixed.

Reply:
It's the only line which is not hard wrapped, might be worth fixing.
But overall it would be better to refer to https://docs.godotengine.org/en/latest/community/contributing/best_practices_for_engine_contributors.html than the less nuanced graph of that earlier blog post IMO.

Reply:
Yes, I personally find the graph discouraging to say the least, talking about https://godotengine.org/article/will-your-contribution-be-merged-heres-how-tell.

While I do understand the rationale, the "We *refuse* by default" statement does not contribute to the open mindedness of open source software development, but again maybe I'm lacking experience to say that.

Else we get "proposals" like #486 with the opposite attitude of "I *propose* by default". 🙂

Reply:
Changed to point to the new doc.